packages/ckeditor5-select-all/tests/selectallcommand.js

```diff
@@ -103,5 +111,59 @@ describe( 'SelectAllCommand', () => {
 				'</table>'
 			);
 		} );
+
+		it( 'consecutive execute() on nested editable, should select all in the parent limit element', () => {
```

Comment:
I would split this to the separate tests to make sure that the next `execute` is expanding selection based on the current selection, and not on some internal state.